### DISCUSSION TITLE
My nativeWind suddenly stopped working, i thing some issue in tailwind config

### DISCUSSION THREAD

**Discussion by ammartiger** [nativewind/nativewind#212] [Q&A] [answered]
My nativeWind suddenly stopped working, i thing some issue in tailwind config. is this file ok. I want to apply that to my components folder as well and its sub files.

![Screenshot_2](https://user-images.githubusercontent.com/86065221/187398280-57939ff2-2f79-4014-945c-10d2866c8d4d.jpg)

...(javascript)
module.exports = {
   content: [
    "./App.{js,jsx,ts,tsx}",
    "./screens/**/*.{js,jsx,ts,tsx}",
   "./components/**/*.{js,jsx,ts,tsx}",
    "./components/.js",
 
  ],
  theme: {
    extend: {},
  },
  plugins: [],
};
...

**Comment by marklawlor** [comment, marked as answer]
That does not look correct. Your have double slashes and parentheses instead of braces

```
"./screens//*.{js,jsx,ts,tsx}",
"./Components//*.{js)"
```

Please refer to the Tailwind docs for more info https://tailwindcss.com/docs/content-configuration

You can also follow the NativeWind troubleshooting guide https://www.nativewind.dev/guides/troubleshooting

**Reply by ammartiger** [reply]
Sorry typo(corrected now) but still its not working in the components folder. once I move my files to screens folder they work fine.
![Screenshot_1](https://user-images.githubusercontent.com/86065221/187397635-ff94411e-96cc-421b-8e6e-b28f2460b43e.jpg)

  
![Screenshot_2](https://user-images.githubusercontent.com/86065221/187398188-778872fc-d21a-45d5-bfcc-167ad67f2bfb.jpg)

**Reply by marklawlor** [reply]
I do not understand what the 4th entry in your contents array is meant to be.

Screenshots are a very slow and painful way to seek help. If the troubleshooting guides do not help, please share a link to your repository.

**Reply by ammartiger** [reply]
Thanks, IT started working after i restarted my PC

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
